packages/wrangler/src/index.ts

```diff
@@ -617,7 +689,7 @@ export function createCLIParser(argv: string[]) {
 	// whoami
 	wrangler.command(
 		"whoami",
-		"🕵️  Retrieve your user info and test your auth config",
+		"🕵️  Retrieve your user information and test your authentication configuration",
```

Comment:
This is pretty long for the terminal now—how about just `🕵️ Retrieve your user information`?

Reply:
SGTM 👍